### PR TITLE
ksync 0.4.7

### PR DESCRIPTION
**Pull request by chenrui333**
[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Bo98** [comment]
Try remove the `go mod tiny` workaround here.

**Comment by carlocab** [comment]
Still broken without the `go mod tidy` workaround.

**Comment by SMillerDev** [comment]
Posted in https://github.com/ksync/ksync/issues/504 again

**Comment by Bo98** [comment]
`0.4.7-hotfix` has been tagged to fix the issue.

File: Formula/ksync.rb [hunk 1]
```diff
@@ -2,8 +2,9 @@ class Ksync < Formula
   desc "Sync files between your local system and a kubernetes cluster"
   homepage "https://ksync.github.io/ksync/"
   url "https://github.com/ksync/ksync.git",
-      tag:      "0.4.6",
-      revision: "bfb445b179a0405ab4a01b999010010406f425b7"
+      tag:      "0.4.7-hotfix",
+      revision: "14ec9e24670b90ee45d4571984e58d3bff02c50e"
+  version "0.4.7" # remove when tag is a proper version
```
Comment (carlocab) [inline Formula/ksync.rb:7]: Oops, maybe this isn't needed. Couldn't hurt, I guess?

Comment (timfallmk) [reply Formula/ksync.rb:7]: `0.4.7` could also removed upstream if that would help avoid this case. I don't want to disrupt anyone's current workflow.

Comment (carlocab) [reply Formula/ksync.rb:7]: No, there's no need for you to do that on our account. Thanks for the offer though!

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**